Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/Settings.java

```diff
@@ -24,6 +24,7 @@
   public static final String FILE_WIIMOTE = "WiimoteNew";
 
   public static final String SECTION_INI_ANDROID = "Android";
+  public static final String SECTION_INI_ANDROID_OVERLAY_BUTTONS = "AndroidOverlayButtons";
```

Comment:
This section name feels to me like it would only include things like button positions and whether each button is enabled, but it also includes things like `MAIN_JOYSTICK_REL_CENTER`. If such other settings are also going to be included, what do you think about calling it "AndroidOverlay" instead?

Reply:
Button position data will occupy a lot of space. I think I'd rather keep the `AndroidOverlayButtons` section and move out settings that don't have the word "button" in them. Some of these settings could fit in either section.